sway-lib-std/src/ecr.sw

```diff
@@ -266,6 +266,18 @@ fn test_ec_recover_r1() {
     assert(public_key.bytes[1] == pub_lo);
 }
 
+#[test(should_revert)]
+fn test_revert_ec_recover_r1() {
+    use ::assert::assert;
+    use ::constants::ZERO_B256;
+    
+    let hi = ZERO_B256;
+    let lo = 0x44ac566bd156b4fc71a4a4cb2655d3da360c695edb27dc3b64d621e122fea23d;
+    let msg_hash = 0x1e45523606c96c98ba970ff7cf9511fab8b25e1bcd52ced30b81df1e4a9c4323;
+    let signature: B512 = B512::from((hi, lo));
+    let public_key = ec_recover_r1(signature, msg_hash).unwrap();
```

Comment:
Why not asserting if the returned error is correct?

Reply:
I've added `should_revert = "0"` but this isn't as intuitive as `should_revert = EcRecoverError::UnrecoverablePublicKey`